### PR TITLE
🌐 Add Polish translation for `docs/pl/docs/index.md`

### PR DESCRIPTION
**Pull request by MicroPanda123**
Index.md to polish translation

### CODE REVIEW

File: docs/pl/docs/index.md [hunk 1]
```diff
@@ -1,12 +1,8 @@
-
-{!../../../docs/missing-translation.md!}
-
-
 <p align="center">
   <a href="https://fastapi.tiangolo.com"><img src="https://fastapi.tiangolo.com/img/logo-margin/logo-teal.png" alt="FastAPI"></a>
 </p>
 <p align="center">
-    <em>FastAPI framework, high performance, easy to learn, fast to code, ready for production</em>
+    <em>FastAPI framework, szybka, prosta i gotowa do użycia w produkcji</em>
```
Comment (dutkiewicz) [inline docs/pl/docs/index.md:5]:
```suggestion
    <em>FastAPI to szybki, prosty i gotowy do użycia w produkcji framework</em>
```

File: docs/pl/docs/index.md [hunk 2]
```diff
@@ -22,31 +18,31 @@
...
-**Documentation**: <a href="https://fastapi.tiangolo.com" target="_blank">https://fastapi.tiangolo.com</a>
+**Dokumentacja**: <a href="https://fastapi.tiangolo.com" target="_blank">https://fastapi.tiangolo.com</a>
 
-**Source Code**: <a href="https://github.com/tiangolo/fastapi" target="_blank">https://github.com/tiangolo/fastapi</a>
+**Kod żródłowy**: <a href="https://github.com/tiangolo/fastapi" target="_blank">https://github.com/tiangolo/fastapi</a>
 
 ---
 
-FastAPI is a modern, fast (high-performance), web framework for building APIs with Python 3.6+ based on standard Python type hints.
+FastAPI to nowoczesny, wydajny, framework sieciowy do budowania API używając Python 3.6+ bazujący na standardowym typowaniu Pythona.
```
Comment (dutkiewicz) [inline docs/pl/docs/index.md:27]:
```suggestion
FastAPI to nowoczesny, wydajny framework webowy do budowania API z użyciem Pythona 3.6+ bazujący na standardowym typowaniu Pythona.
```

File: docs/pl/docs/index.md [hunk 3]
```diff
@@ -22,31 +18,31 @@
...
 
-The key features are:
+Cechami kluczowymi są:
 
-* **Fast**: Very high performance, on par with **NodeJS** and **Go** (thanks to Starlette and Pydantic). [One of the fastest Python frameworks available](#performance).
+* **Wydajność**: FastApi jest bardzo wydajny, na równi z **NodeJS** oraz **Go** (podziękowania dla Starlette i Pydantic). [Jedna z najszybszych frameworków Pythonowych dostępnych](#wydajność).
```
Comment (dutkiewicz) [inline docs/pl/docs/index.md:31]:
```suggestion
* **Wydajność**: FastApi jest bardzo wydajny, na równi z **NodeJS** oraz **Go** (podziękowania dla Starlette i Pydantic). [Jeden z najszybszych dostępnych frameworków Pythonowych ](#wydajność).
```

File: docs/pl/docs/index.md [hunk 4]
```diff
@@ -22,31 +18,31 @@
...
 
-* **Fast to code**: Increase the speed to develop features by about 200% to 300%. *
-* **Fewer bugs**: Reduce about 40% of human (developer) induced errors. *
-* **Intuitive**: Great editor support. <abbr title="also known as auto-complete, autocompletion, IntelliSense">Completion</abbr> everywhere. Less time debugging.
-* **Easy**: Designed to be easy to use and learn. Less time reading docs.
-* **Short**: Minimize code duplication. Multiple features from each parameter declaration. Fewer bugs.
-* **Robust**: Get production-ready code. With automatic interactive documentation.
-* **Standards-based**: Based on (and fully compatible with) the open standards for APIs: <a href="https://github.com/OAI/OpenAPI-Specification" class="external-link" target="_blank">OpenAPI</a> (previously known as Swagger) and <a href="https://json-schema.org/" class="external-link" target="_blank">JSON Schema</a>.
+* **Szybkość napisania**: Przyśpiesz czas pisania funkcji o około 200% do 300%. *
+* **Mniejsza ilość błędów**: Zmniejsz błędy ludzkie (dewelopera) o około 40%. *
```
Comment (dutkiewicz) [inline docs/pl/docs/index.md:34]:
```suggestion
* **Mniejsza ilość błędów**: Zmniejsz ludzkie (dewelopera) błędy o około 40%. *
```

File: docs/pl/docs/index.md [hunk 5]
```diff
@@ -22,31 +18,31 @@
...
+* **Intuicyjność**: Wspaniałe wsparcie przez edytory kodu. Wszędzie <abbr title="znane jako auto-complete, autocompletion, IntelliSense">automatyczne uzupełnianie</abbr>. Mniejszy czas debugowania.
+* **Łatwość**: Zaprojektowana by być prosta i łatwa do nauczenia. Mniej czasu spędzanego na czytanie dokumentacji.
```
Comment (dutkiewicz) [inline docs/pl/docs/index.md:36]:
```suggestion
* **Łatwość**: Zaprojektowany by być prosty i łatwy do nauczenia. Mniej czasu spędzaj na czytanie dokumentacji.
```

File: docs/pl/docs/index.md [hunk 6]
```diff
@@ -22,31 +18,31 @@
...
+* **Krótkość**: Zmniejsz powtarzanie sie kodu. Dla każdego parametru można dać wiele funkcji. Mniej błędów.
```
Comment (dutkiewicz) [inline docs/pl/docs/index.md:37]:
```suggestion
* **Krótkość**: Zmniejsz powtarzanie się kodu. Dla każdego parametru można dać wiele funkcji. Mniej błędów.
```

File: docs/pl/docs/index.md [hunk 7]
```diff
@@ -22,31 +18,31 @@
...
+* **Solidność**: Kod może być stosowany w produkcji. Wraz z automatyczną interaktywną dokumentacją.
```
Comment (dutkiewicz) [inline docs/pl/docs/index.md:38]:
```suggestion
* **Solidność**: Kod gotowy do środowiska produkyjnego. Wraz z automatyczną interaktywną dokumentacją.
```

File: docs/pl/docs/index.md [hunk 8]
```diff
@@ -22,31 +18,31 @@
...
+* **Bazowane na standardach**: Bazowane na (i w pełni kompatybilne z) otwartymi standardami w API: <a href="https://github.com/OAI/OpenAPI-Specification" class="external-link" target="_blank">OpenAPI</a> (wcześniej znane jako Swagger) oraz <a href="https://json-schema.org/" class="external-link" target="_blank">JSON Schema</a>.
```
Comment (dutkiewicz) [inline docs/pl/docs/index.md:39]:
```suggestion
* **Oparte na standardach**: Oparte na (i w pełni kompatybilne z) otwartymi standardami w API: <a href="https://github.com/OAI/OpenAPI-Specification" class="external-link" target="_blank">OpenAPI</a> (wcześniej znane jako Swagger) oraz <a href="https://json-schema.org/" class="external-link" target="_blank">JSON Schema</a>.
```

File: docs/pl/docs/index.md [hunk 9]
```diff
@@ -22,31 +18,31 @@
...
 
-<small>* estimation based on tests on an internal development team, building production applications.</small>
+<small>* oszacowania bazowane na testach przez wewnętrzny zespół deweloperów, budujących aplikacie używane w produkcji.</small>
```
Comment (dutkiewicz) [inline docs/pl/docs/index.md:41]:
```suggestion
<small>* oszacowania bazowane na testach przez wewnętrzny zespół deweloperów, budujących aplikacie używane na środowisku produkcyjnym.</small>
```

File: docs/pl/docs/index.md [hunk 10]
```diff
@@ -101,24 +97,24 @@ The key features are:
...
-**Typer** is FastAPI's little sibling. And it's intended to be the **FastAPI of CLIs**. ⌨️ 🚀
+**Typer** to młodsze rodzeństwo FastAPI. I jego przeznaczenie to bycie **FastAPI aplikacji konsolowych** . ⌨️ 🚀
 
-## Requirements
+## Wymagania
 
 Python 3.6+
 
-FastAPI stands on the shoulders of giants:
+FastAPI stoi na grzbiecie gigantów:
```
Comment (dutkiewicz) [inline docs/pl/docs/index.md:112]:
```suggestion
FastAPI stoi na ramionach olbrzymów:
```

File: docs/pl/docs/index.md [hunk 11]
```diff
@@ -142,11 +138,11 @@ $ pip install uvicorn[standard]
 
 </div>
 
-## Example
+## Przykład
 
-### Create it
+### Napisz
```
Comment (dutkiewicz) [inline docs/pl/docs/index.md:143]:
```suggestion
### Stwórz
```

File: docs/pl/docs/index.md [hunk 12]
```diff
@@ -214,54 +210,53 @@ INFO:     Application startup complete.
...
+* Otrzymuje żądanie HTTP w _ścieżce_ `/` i `/items/{item_id}`.
+* Obie _ścieżki_ używają <em>operacji</em> `GET` (znane także jako _metody_ HTTP).
+* _Ścieżka_ `/items/{item_id}` ma _parametr ścieżki_ `item_id` który powinien być obiektem typu `int`.
+* _Ścieżka_ `/items/{item_id}` ma opcjonalny _parametr zapytania_ typu `str` o nazwie `q`.
 
-### Interactive API docs
+### Interaktywna dokumentacja API
 
-Now go to <a href="http://127.0.0.1:8000/docs" class="external-link" target="_blank">http://127.0.0.1:8000/docs</a>.
+Teraz wejdz na strone <a href="http://127.0.0.1:8000/docs" class="external-link" target="_blank">http://127.0.0.1:8000/docs</a>.
```
Comment (dutkiewicz) [inline docs/pl/docs/index.md:241]:
```suggestion
Teraz wejdź na strone <a href="http://127.0.0.1:8000/docs" class="external-link" target="_blank">http://127.0.0.1:8000/docs</a>.
```

File: docs/pl/docs/index.md [hunk 13]
```diff
@@ -214,54 +210,53 @@ INFO:     Application startup complete.
...
 ![ReDoc](https://fastapi.tiangolo.com/img/index/index-02-redoc-simple.png)
 
-## Example upgrade
+## Aktualizacja przykładu
 
-Now modify the file `main.py` to receive a body from a `PUT` request.
+Teraz zmodyfikujemy plik `main.py` aby otrzmywał treść (body, w przykładzie nazwane `Item`) żądania `PUT`.
 
-Declare the body using standard Python types, thanks to Pydantic.
+Zadeklaruj treść żądania używając standardowych typów w Pythonie, zasługi dla Pydantic.
```
Comment (dutkiewicz) [inline docs/pl/docs/index.md:259]:
```suggestion
Zadeklaruj treść żądania, używając standardowych typów w Pythonie dzięki Pydantic.
```

File: docs/pl/docs/index.md [hunk 14]
```diff
@@ -293,175 +288,175 @@ def update_item(item_id: int, item: Item):
...
+* Konwersuje z i do JSON automatycznie.
+* Dokumentuje wszystko z OpenAPI, które może być używane przez:
+    * Interaktywne systemy dokumentacji.
+    * Systemy automatycznego generowania kodu klientowego, dla wielu języków.
+* Dostarczy bezpośrednio 2 interaktywne dokumentacje sieciowe.
 
 ---
 
-We just scratched the surface, but you already get the idea of how it all works.
+To dopiero początek, ale już masz mniejwięcej pojęcie jak to wszystko działa.
```
Comment (dutkiewicz) [inline docs/pl/docs/index.md:388]:
```suggestion
To dopiero początek, ale już masz mniej-więcej pojęcie jak to wszystko działa.
```

File: docs/pl/docs/index.md [hunk 15]
```diff
@@ -293,175 +288,175 @@ def update_item(item_id: int, item: Item):
...
-* Security and authentication, including support for **OAuth2** with **JWT tokens** and **HTTP Basic** auth.
-* More advanced (but equally easy) techniques for declaring **deeply nested JSON models** (thanks to Pydantic).
-* Many extra features (thanks to Starlette) as:
-    * **WebSockets**
+* Deklaracje **parametrów** z innych miejsc takich jak: **headery**, **pliki cookies**, **formularze** i **pliki**.
+* Jak ustawić **ograniczenia walidacyjne** takie jak `maksymalna długość` lub `regex`.
+* Potężny i łatwy w użyciu system **<abbr title="znane jako komponenty, resources, providers, services, injectables">Dependency Injection</abbr>**.
+* Zabezpieczenia i autoryzacja, wliczając wsparcie dla **OAuth2** z **tokenami JWT** oraz autoryzacją **HTTP Basic**.
+* Bardziej zaawansowane (ale równie proste) techniki deklarowania **głęboko zagnieżdżonych modeli JSON** (podziękowania dla Pydantic).
+* Wiele dodatkowych funkcji (podziękowania dla Starlette) takie jak:
```
Comment (dutkiewicz) [inline docs/pl/docs/index.md:421]:
```suggestion
* Wiele dodatkowych funkcji (dzięki Starlette) takie jak:
```

**Review by dutkiewicz — CHANGES_REQUESTED** [review]

**Comment by dutkiewicz** [comment]
Fixed automatic translation to more grammatically correct with the Polish language

**Comment by dutkiewicz** [comment]
No worries, that's the point of pull request reviews :) Ping me if you want to translate other docs

**Review by dutkiewicz — APPROVED** [review]

**Comment by bxsx** [comment]
Hey @MicroPanda123 
Just stumbled upon this PR and believe it still needs a lot of improvement (spelling, grammar, inaccurate terminology, etc.). Would you mind granting me write access to your fork? I can help with the translation, but there are too many issues here to play with GitHub suggestions. Thanks!

**Comment by MicroPanda123** [comment]
Hey @bxsx, done, I gave you direct access, thanks for help! I appreciate it.

**Comment by bxsx** [comment]
I've pushed f6159a8 with all my changes.

@MicroPanda123 @dutkiewicz would you like to take a look and approve?

**Comment by MicroPanda123** [comment]
It looks good to me.

File: docs/pl/docs/index.md [hunk 16]
```diff
@@ -22,29 +18,28 @@
...
-**Documentation**: <a href="https://fastapi.tiangolo.com" target="_blank">https://fastapi.tiangolo.com</a>
+**Dokumentacja**: <a href="https://fastapi.tiangolo.com" target="_blank">https://fastapi.tiangolo.com</a>
 
-**Source Code**: <a href="https://github.com/tiangolo/fastapi" target="_blank">https://github.com/tiangolo/fastapi</a>
+**Kod żródłowy**: <a href="https://github.com/tiangolo/fastapi" target="_blank">https://github.com/tiangolo/fastapi</a>
 
 ---
 
-FastAPI is a modern, fast (high-performance), web framework for building APIs with Python 3.6+ based on standard Python type hints.
+FastAPI to nowoczesny, wydajny framework webowy do budowania API z użyciem Pythona 3.6+ bazujący na standardowym typowaniu Pythona.
```
Comment (dimaqq) [inline docs/pl/docs/index.md:27]:
```suggestion
FastAPI to nowoczesny, wydajny framework webowy do budowania API z użyciem Pythona 3.6+ bazujący się na standardowym typowaniu Pythona.
```
I think?

**Review by dimaqq — COMMENTED** [review]
my 2c: this translation is 10x better than no translation at all!

**Review by dimaqq — APPROVED** [review]

File: docs/pl/docs/index.md [hunk 17]
```diff
@@ -24,7 +24,7 @@
 
 ---
 
-FastAPI to nowoczesny, wydajny framework webowy do budowania API z użyciem Pythona 3.6+ bazujący na standardowym typowaniu Pythona.
+FastAPI to nowoczesny, wydajny framework webowy do budowania API z użyciem Pythona 3.6+ bazujący się na standardowym typowaniu Pythona.
```
Comment (dimaqq) [inline docs/pl/docs/index.md:27]: Huh, I ran my suggestion by a native speaker and she says it should be `framework ... bazujący na ... typowaniu`. Without `się`.

So sorry I led you astray 🙈

Comment (MicroPanda123) [reply docs/pl/docs/index.md:27]: Don't worry, mistakes happen.

Comment (bxsx) [reply docs/pl/docs/index.md]: Reverted

**Comment by tiangolo** [comment]
Thank you @MicroPanda123! :rocket:

And thanks for all the help @bxsx, @dimaqq, and @dutkiewicz! :bow: :cake:

**Pull request merged**